Apps/W1/HybridGP/app/src/pages/GPCompanyMigrationSettings.Page.al

```diff
@@ -0,0 +1,155 @@
+page 40056 "GP Company Migration Settings"
```

Comment:
I believe this object should be added to the permission sets, together with the new query

Reply:
Updated, thanks